criu/tty.c

```diff
@@ -1977,7 +1977,10 @@ static int dump_one_tty(int lfd, u32 id, const struct fd_parms *p)
 	pr_info("Dumping tty %d with id %#x\n", lfd, id);
 
 	driver = get_tty_driver(p->stat.st_rdev, p->stat.st_dev);
-	if (driver->fd_get_index)
+	if (driver == NULL) {
+		pr_err("Unable to find a tty driver (rdev %#lx dev %#lx)\n", p->stat.st_rdev, p->stat.st_dev);
+		return -1;
+	} else if (driver->fd_get_index)
```

Comment:
There is no need to add `else` here because `return -1;` would prevent the execution of subsequent instructions.